website/src/documentation/sdks/java-dependencies.md

```diff
@@ -26,336 +26,51 @@ behavior in the service. If you are using any of these packages in your code, be
-  <tr><td>com.fasterxml.jackson.core</td><td>jackson-databind</td><td>2.9.5</td></tr>
-  <tr><td>com.fasterxml.jackson.dataformat</td><td>jackson-dataformat-cbor</td><td>2.9.5</td></tr>
-  <tr><td>com.fasterxml.jackson.dataformat</td><td>jackson-dataformat-yaml</td><td>2.9.5</td></tr>
-  <tr><td>com.fasterxml.jackson.datatype</td><td>jackson-datatype-joda</td><td>2.9.5</td></tr>
-  <tr><td>com.fasterxml.jackson.module</td><td>jackson-module-scala_2.11</td><td>2.9.5</td></tr>
-  <tr><td>javax.xml.bind</td><td>jaxb-api</td><td>2.2.12</td></tr>
-  <tr><td>joda-time</td><td>joda-time</td><td>2.4</td></tr>
-  <tr><td>junit</td><td>junit</td><td>4.12</td></tr>
-  <tr><td>org.apache.kafka</td><td>kafka_2.11</td><td>1.0.0</td></tr>
-  <tr><td>org.apache.kafka</td><td>kafka-clients</td><td>1.0.0</td></tr>
-  <tr><td>org.apache.apex</td><td>malhar-library</td><td>3.4.0</td></tr>
-  <tr><td>org.mockito</td><td>mockito-core</td><td>1.10.19</td></tr>
-  <tr><td>io.netty</td><td>netty-handler</td><td>4.1.25.Final</td></tr>
-  <tr><td>io.netty</td><td>netty-tcnative-boringssl-static</td><td>2.0.8.Final</td></tr>
-  <tr><td>io.netty</td><td>netty-transport-native-epoll</td><td>4.1.25.Final</td></tr>
-  <tr><td>org.postgresql</td><td>postgresql</td><td>42.2.2</td></tr>
-  <tr><td>org.powermock</td><td>powermock-mockito-release-full</td><td>1.6.4</td></tr>
-  <tr><td>com.google.protobuf</td><td>protobuf-java</td><td>3.6.0</td></tr>
-  <tr><td>com.google.protobuf</td><td>protobuf-java-util</td><td>3.6.0</td></tr>
-  <tr><td>com.google.api.grpc</td><td>proto-google-cloud-pubsub-v1</td><td>1.18.0</td></tr>
-  <tr><td>com.google.api.grpc</td><td>proto-google-cloud-spanner-admin-database-v1</td><td>0.19.0</td></tr>
-  <tr><td>com.google.api.grpc</td><td>proto-google-common-protos</td><td>1.12.0</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-api</td><td>1.7.25</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-jdk14</td><td>1.7.25</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-log4j12</td><td>1.7.25</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-simple</td><td>1.7.25</td></tr>
-  <tr><td>org.xerial.snappy</td><td>snappy-java</td><td>1.1.4</td></tr>
-  <tr><td>org.apache.spark</td><td>spark-core_2.11</td><td>2.3.2</td></tr>
-  <tr><td>org.apache.spark</td><td>spark-network-common_2.11</td><td>2.3.2</td></tr>
-  <tr><td>org.apache.spark</td><td>spark-streaming_2.11</td><td>2.3.2</td></tr>
-  <tr><td>org.codehaus.woodstox</td><td>stax2-api</td><td>3.1.4</td></tr>
-  <tr><td>org.codehaus.woodstox</td><td>woodstox-core-asl</td><td>4.4.1</td></tr>
-  <tr><td>com.pholser</td><td>junit-quickcheck-core</td><td>0.8</td></tr>
-</table>
+    Replace `<VERSION_NUMBER>` with the major.minor.patch version of the SDK. For example, <https://raw.githubusercontent.com/apache/beam/v2.18.0/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy>{:target="_blank"} will provide the dependencies for the 2.18.0 release.
```

Comment:
Can we use `{{ site.release_latest }}` for the example as well? Same for the Python guide.

Reply:
Yup, done.